openpype/hosts/aftereffects/plugins/load/load_file.py

```diff
@@ -31,13 +31,8 @@ def load(self, context, name=None, namespace=None, data=None):
 
         path = self.filepath_from_context(context)
 
-        repr_cont = context["representation"]["context"]
-        if "#" not in path:
-            frame = repr_cont.get("frame")
-            if frame:
-                padding = len(frame)
-                path = path.replace(frame, "#" * padding)
-                import_options['sequence'] = True
+        if isinstance(context["representation"].get("files"), list):
```

Comment:
I think that `"files"` on representation is always list.

Reply:
Previous implementation was correct.
```suggestion
        if len(context["representation"]["files"]) > 1:
```

Reply:
Thanks for confirming - [I wasn't sure in this comment](https://github.com/ynput/OpenPype/pull/5581#discussion_r1317565835)